### PR TITLE
Editing:x

### PR DESCRIPTION
**Pull request by avalanche2121**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,6 @@ This is a playground for those learning Git.
 
 This [Git visualizer](https://git-school.github.io/visualizing-git/) is
 also a great learning tool.
+
+
+Edit
```
Comment (danielyli) [inline README.md]: Nice, but can you please be more descriptive? Perhaps a description of what the edit is would help…

File: README.md [hunk 2]
```diff
@@ -4,3 +4,6 @@ This is a playground for those learning Git.
 
 This [Git visualizer](https://git-school.github.io/visualizing-git/) is
 also a great learning tool.
+
+
```
Comment (danielyli) [inline README.md]: No need for two blank lines when one will do :-)

File: README.md (hunk 1 above)
Comment (avalanche2121) [reply README.md]: Acknowledged.

File: README.md (hunk 2 above)
Comment (avalanche2121) [reply README.md]: Acknowledged.

**Pull request merged**